### ISSUE TITLE
The performance on city8.res50v3+.CPS 

### ISSUE THREAD

**Issue by HYheyue** [charlesCXK/TorchSemiSeg#43]
Hello, I trained labeled ratio  8,  137 epochs, learning_rate  0.02, batch size 8 with 8 cards as your default configuration in city8.res50v3+.CPS，but the mean_IU is 70.682% which is lower than 74.39% in the paper. Later，i change the epoch to 240 and the learning rate to 0.04 refers to the experiments in the paper，but it still doesn't work. Do you have any advice for it？

Thank you a lot.

**Comment by charlesCXK** [comment]
Hi, we recommend using the docker environment we provide to reproduce the results, for that it has exactly the same environment as us.

**Comment by HYheyue** [comment]
Thank you for your reply，but the config 137 epochs is enough for cityscapes？ Should I take the paper or the code as the standard？

**Comment by charlesCXK** [comment]
The code is exactly the same as that of the paper. If you are confused about why we use 137 epochs, please refer to https://github.com/charlesCXK/TorchSemiSeg/blob/main/docs/getting_started.md for details.

**Comment by HYheyue** [comment]
I understand what you mean about the config. 
But there is a new problem, I set trade-off weight lambda to 0 to remove the cross pseudo supervision loss, and the mean_IU is 70.958% which is higher than when the lambda is 6. This shows that the  cross pseudo supervision loss is useless, does it also show that it has nothing to do with the environmental impact?
We still can’t reproduce your performance and hope to get your help!

**Comment by Luffy03** [comment]
> I understand what you mean about the config. But there is a new problem, I set trade-off weight lambda to 0 to remove the cross pseudo supervision loss, and the mean_IU is 70.958% which is higher than when the lambda is 6. This shows that the cross pseudo supervision loss is useless, does it also show that it has nothing to do with the environmental impact? We still can’t reproduce your performance and hope to get your help!

Hi, I got similar problems. Have you figured it out?

**Comment by alittledou** [comment]
> 

Excuse me,how long does it take you to train a model?
